lib/hexpm_web/controllers/controller_helpers.ex

```diff
@@ -325,6 +325,10 @@ defmodule HexpmWeb.ControllerHelpers do
   def auth_error_message(:unconfirmed),
     do: "Email has not been verified yet. You can resend the verification email below."
 
+  def password_breached_message(),
+    do: "The password you provided has previously been breached.
+      To increase your security, please change your password."
```

Comment:
I think this is a little bit too terse; wdyt about at least adding something like "according to https://haveibeenpwned.com"? Or instead maybe we'd document this check in docs (mention stuff like what is havebeenpwned.com, that we don't send passwords in plaintext to them etc) and link to that docs page from the flash message? I'm happy to help with copy for the docs if we're gonna move in that direction.

Reply:
If we added a link to the docs/faqs in the flash message perhaps we could be a lot more expressive as to what this message is all about.
This looks similar to what GitHub does:
<img width="1679" alt="pwned" src="https://user-images.githubusercontent.com/773380/58750566-4bc61180-8451-11e9-8af3-04f6eeba5cda.png">